### DISCUSSION TITLE
What is the default value for "setstat_mode"?

### DISCUSSION THREAD

**Discussion by derekmahar** [drakkan/sftpgo#897] [Q&A] [answered]
What is the default value for common [configuration](https://github.com/drakkan/sftpgo/blob/719f6077ab4c0ba569ae2f5a0a601d30c6d49cce/docs/full-configuration.md) option `setstat_mode`?

**Comment by drakkan** [comment, marked as answer]
You can find default values in the default [config file](https://github.com/drakkan/sftpgo/blob/main/sftpgo.json)

**Reply by derekmahar** [reply]
Thank you. Presently, the default value of `setstat_mode` in the default config file is 0.

**Reply by drakkan** [reply]
> Thank you. Presently, the default value of `setstat_mode` in the default config file is 0.

The configuration changes over time and sometimes I forgot to update the docs. The default configuration file can be used as a reference
